### ISSUE TITLE
Standardize cities naming convention

### ISSUE THREAD

**Issue by ustroetz** [mapzen/metroextractor-cities#227]
US cities are missing the country name. I understand that some cities in the US have the same name, like `portland_oregon` and `portland_maine`, but I think the naming should still be consistent and we should add the country too.

My suggestion is to use `name_state_country`. While state can be optional. Any thoughts on that?

**Comment by hkrishna** [comment]
Yes, I think adding ```country``` to all the names is a good idea. Its always nice to have a consistent naming convention. Maybe we could use [alpha3](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) values for countries? 

And ```state``` could be optional - we only use it for cases like ```portland``` - ```portland_oregon_usa``` ```portland_maine_usa```

**Comment by louh** [comment]
@thatdatabaseguy is generating a manifest for OpenVenues datasets that contains metadata fields, including country name. I'm not sure what's generating it, but here is the result: http://s3.amazonaws.com/openvenues-static-public/cities/manifest.json

I believe the boundaries for OpenVenues is now using the same bboxes as the Metro Extractor, so we have a common interest on this.

**Comment by souperneon** [comment]
Hi Uli,

At Mapzen, we just introduced an on-demand feature to make it easier and faster get custom extracts of places all over the world. [Give it a try](https://mapzen.com/data/metro-extracts-alt/)!

We are still testing it with people and would love it if you could give us your feedback by participating in a short usability study for the new experience.  [Let us know](http://goo.gl/forms/OIEdDWr7RIC04oqo2)! You can also send us your feedback anytime at [hello@mapzen.com](mailto:hello@mapzen.com)

Thanks,
Ekta.

**Comment by ustroetz** [comment]
Wow that's great @souperneon! Very excited to check it out.
